packages/odie-client/src/components/ellipsis-menu/index.tsx

```diff
@@ -10,17 +11,22 @@ import './style.scss';
 type EllipsisMenuProps = {
 	position?: string;
 	popoverClassName?: string;
+	trackEventProps?: { source: string };
 } & PropsWithChildren;
 
 export const EllipsisMenu: FunctionComponent< EllipsisMenuProps > = ( {
 	position,
 	children,
 	popoverClassName,
+	trackEventProps,
 } ) => {
 	const [ isMenuVisible, setMenuVisible ] = useState( false );
 	const popoverContext = useRef< HTMLButtonElement >( null );
 
 	const handleClick = () => {
+		if ( ! isMenuVisible ) {
+			recordTracksEvent( 'calypso_help_open_ellipsis_menu', trackEventProps );
+		}
```

Comment:
I pushed a change, since this was using `trackEvents` from Odie context, but this Ellipsis Menu is used in the Help Center and so it has not the context.